### ISSUE TITLE
LibreOffice Writer: Unreadable dialog hyphenation

### ISSUE THREAD

**Issue by peterpacult** [nvaccess/nvda#11689]
### Steps to reproduce:
1. Open Writer
2. Write a few sentences.
3. Open the hyphenation dialog.
4. At a point, a dialog appears with the options "Yes" or "No".
5. For users working with sscreenreaders, it is not clear, what the dialog is asking.

### Actual behavior:
The dialog is not readable for users working withc screenreaders.

### Expected behavior:
The screenreader should be announcing, what this dialog is about.

#### NVDA installed/portable/running from source:
installed
#### NVDA version:
2020.2

#### Windows version:
Edition	Windows 	10 Home
Version				1909
Operating system build	18363.1016

#### Name and version of other software in use when reproducing the issue:
LibreOffice
Version:	7.0.0.3 (x64)
Build ID:	8061b3e9204bef6b321a21033174034a5e2ea88e
CPU-Threads: 	4; BS: Windows 10.0 Build 18363; UI-Render: Skia/Vulkan; VCL: win
Locale: 	de-DE (de_DE); UI: de-DE
Calc: 	threaded

#### Other information about your system:
---
#### Does the issue still occur after restarting your computer?
Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No.

#### If addons are disabled, is your problem still occuring?
Not tested.

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
Not tested.

**Comment by peterpacult** [comment]
This issue has also been reported to the Document Foundation; see https://bugs.documentfoundation.org/show_bug.cgi?id=135770

**Comment by michaelweghorn** [comment]
I just retested this. The text ("Continue checking at beginning of document") is announced just fine now.

This was a general issue that message dialogs in LibreOffice were not announced and was fixed by efb614193ee9bf041570b363f654ace7cc6d834c, so this issue can IMHO be closed now (as a duplicate of #11687).

**Comment by michaelweghorn** [comment]
> I just retested this. The text ("Continue checking at beginning of document") is announced just fine now.
> 
> This was a general issue that message dialogs in LibreOffice were not announced and was fixed by [efb6141](https://github.com/nvaccess/nvda/commit/efb614193ee9bf041570b363f654ace7cc6d834c), so this issue can IMHO be closed now (as a duplicate of #11687).

And in addition, there were changes in LibreOffice as well, s. https://bugs.documentfoundation.org/show_bug.cgi?id=135588 for details.

@Qchristensen Can you possibly close this issue if you agree with the above? (I don't seem to be able to do that myself.)

**Comment by Adriani90** [comment]
Confirming this works as expected with NVDA 2023.1 RC1 and Libreoffice 7.5.1. Closing as works for me.

**Comment by Adriani90** [comment]
@michaelweghorn thank you very much for fixing this.
